test/e2e/handler/utils.go

```diff
@@ -493,6 +497,11 @@ func kubectlAndCheck(command ...string) {
 	Expect(err).ShouldNot(HaveOccurred(), out)
 }
 
+func kubectlAndCheckOccurred(command ...string) {
```

Comment:
I don't think it is used.. 
and if it is, I'm not sure why we expect the err to occur

Reply:
Ops, yes, thanks for noticing